src/tendencies/vertical_diffusion_boundary_layer.jl

```diff
@@ -42,22 +42,42 @@ function vertical_diffusion_boundary_layer_cache(
     cond_type = NamedTuple{(:shf, :lhf, :E, :ρτxz, :ρτyz), NTuple{5, FT}}
     surface_normal = Geometry.WVector.(ones(axes(Fields.level(Y.c, 1))))
 
+    sfc_conditions = similar(Fields.level(Y.f, half), SF.SurfaceFluxConditions{FT})
+    ts_type = CA.thermo_state_type(Y.c, FT)
+
+    # TODO: replace with (TODO add support for) Base.zero / similar
     surface_scheme_params = if surface_scheme isa BulkSurfaceScheme
-        (;
-            Cd = ones(axes(Fields.level(Y.f, half))) .* FT(0.0044),
-            Ch = ones(axes(Fields.level(Y.f, half))) .* FT(0.0044),
-        )
+
+        sfc_inputs_type = typeof(SF.Coefficients{FT}(;
+            state_in = SF.InteriorValues(FT(0), (FT(0), FT(0)), ts_type),
+            state_sfc = SF.SurfaceValues(FT(0), (FT(0), FT(0)), ts_type),
+            z0m = FT(0),
+            z0b = FT(0),
+            Cd = FT(0),
+            Ch = FT(0),
+        ))
+        sfc_inputs = similar(Fields.level(Y.f, half), sfc_inputs_type)
+        fill!(sfc_inputs.Cd, FT(0.0044))
```

Comment:
This is not urgent, but it would be better to be able to specify this value in the command line arguments.